Code/GraphMol/MolDraw2D/test1.cpp

```diff
@@ -3167,8 +3166,41 @@ void test20Annotate() {
                           " style='font-size:12px;font-style:normal;"
                           "font-weight:normal;fill-opacity:1;stroke:none;"
                           "font-family:sans-serif;text-anchor:start;"
-                          "fill:#000000' >f</text>") !=
+                          "fill:#000000' >f</text>") != std::string::npos);
+#endif
+  }
+  {
+    auto m1 = "S=C1N=C(NC(CC#N)(C)C=C=C)NC2=NNN=C21"_smiles;
+#ifdef RDK_BUILD_CAIRO_SUPPORT
+    {
+      MolDraw2DCairo drawer(200, 200);
+      drawer.drawMolecule(*m1);
+      drawer.drawOptions().addAtomIndices = true;
```

Comment:
The ordering of these two lines should be reversed
```suggestion
      drawer.drawOptions().addAtomIndices = true;
      drawer.drawMolecule(*m1);
```